stl/inc/ranges

```diff
@@ -620,6 +620,132 @@ namespace ranges {
         /* [[no_unique_address]] */ _Ty _Value{};
     };
 
+    // CLASS TEMPLATE ranges::_Non_propagating_cache
+    template <class _Ty>
+    concept _Objectable = is_object_v<_Ty>;
+
+    // clang-format off
+    template <_Objectable _Ty>

```

Comment:
This constraint should be `_Destructible_object`. (For bonus points, the pre-existing `_Copy_constructible_object` should refine `_Destructible_object`.)

Reply:
I understand you comment as renaming `_Objectable` to `_Destructible_object`